release/preview/alpine310/test-deps/docker/Dockerfile

```diff
@@ -8,12 +8,10 @@ FROM node:14.3.0-alpine as node
 
 FROM ${BaseImage}
 
-ARG fromTag=3.9
-
 ENV NODE_VERSION=14.3.0 \
     YARN_VERSION=1.22.4 \
     NVM_DIR="/root/.nvm" \
-    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-TestDeps-Alpine-${fromTag}
+    POWERSHELL_DISTRIBUTION_CHANNEL=PSDocker-TestDeps-Alpine-3.9
```

Comment:
3.9 has been erased

Reply:
What do you mean by "has been erased" exactly? I see there are images on docker hub available for alpine 3.9...do you mean we're just not supporting that anymore?

Reply:
Not just us, but also Alpine itself - 3.9 is EOL'd and I removed it a few days ago in a PR.

Reply:
Ah I see, you're right it's going to EOL in early January. Thanks for the explanation :)